app/views/dashboard/_eeo_data.haml

```diff
@@ -0,0 +1,6 @@
+= form_tag eeo_data_application_records_path, method: :get do
+  List EEO data between
+  = text_field_tag :eeo_start_date, '', class: 'datepicker', required: true
```

Comment:
I think it's also fine (and more consistent with other forms) to just call these fields `start_date` and `end_date`